PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,17 @@
-**Describe the pull request**
+**Note**
+Pull requests should follow the git flow described in the [wiki](https://github.com/CypherpunkArmory/UserLAnd/wiki/Git-flow).
+To get git history clean and aid in more useful release notes, it is essential to use the `Squash and merge` option when
+merging into master, and the `Create merge commit` option.
```

Comment:
Incomplete statement... should say "when merging into release"

Reply:
Yep, it should indeed.